test/BUILD

```diff
@@ -1,31 +1,33 @@
-package(default_visibility=["//visibility:public"])
+package(default_visibility = ["//visibility:public"])
+
 load("@build_bazel_rules_typescript//:defs.bzl", "ts_library")
 load("@build_bazel_rules_nodejs//:defs.bzl", "jasmine_node_test")
 
 ts_library(
     name = "test_support",
     srcs = ["test_support.ts"],
-    deps = ["//src"],
     tsconfig = "//:tsconfig.json",
+    deps = ["//src"],
```

Comment:
curious, did you run a formatter here? I think buildifier is public but I never hooked it up anywhere

Reply:
My emacs autoformats BUILD files, yeah.